test/infer_rt_test.py

```diff
@@ -128,22 +129,22 @@ def test_med_scale_strong_growth_and_decay():
     check_standard_assertions(rt1, rt2, t_switch, rt)
 
 
-@pytest.mark.slow
-def test_low_cases_weak_growth():
-    """Track with low scale (count = 5) and slow growth"""
-    (rt1, rt2, t_switch, rt) = run_individual(
-        "50",  # Vermont
-        load_data.DataSpec(
-            generator_type=load_data.DataGeneratorType.EXP,
-            disable_deaths=True,
-            scale=5.0,
-            ratechange1=RateChange(0, 1.0),
-            ratechange2=RateChange(70, 1.2),
-        ),
-        "test_low_cases_weak_growth",
-    )
-    # check_standard_assertions(rt1, rt2, t_switch, rt)
-    # TODO is failing this test rt = .84 instead of rt1
+# @pytest.mark.slow
```

Comment:
Instead of commenting out this test, can you add a skip? https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

Reply:
Done. The note is vague because Alex had commented out the line in the test that was the actual assertion. So I don't know specifically why it's not passing, just that it the test was passing without any actual test happening.